modules/snpsift/split/meta.yml

```diff
@@ -0,0 +1,43 @@
+name: snpsift_split
+description: Splits a VCF file into chromosomes
+keywords:
+  - split
+  - vcf
+tools:
+  - snpsift:
+      description: SnpSift is a toolbox that allows you to filter and manipulate annotated files
+      homepage: https://pcingola.github.io/SnpEff/ss_introduction/
+      documentation: https://pcingola.github.io/SnpEff/ss_introduction/
+      tool_dev_url: https://github.com/pcingola/SnpEff
+      doi: "10.3389/fgene.2012.00035"
+      licence: ['MIT']
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - vcf:
+      type: file
```

Comment:
https://pcingola.github.io/SnpEff/ss_split/ as far as I understand it could also be a list of vcf files, right?

Reply:
I tested this and apparently you can only split one vcf file at a time

Reply:
A my bad I suppose when you use -j it should work. Since that would be an `args` thing I think we should still add it here like (list) of files

Reply:
so yes, with `Snpsift split` you can only split one vcf file but if you really want to join several vcf files, then you're right that you'd need the `-j` option.

Reply:
Should i also incorporate that option and also a test for the `-j` option then?

Reply:
And if you do use the join option, then it's actually possible to create a named output